.travis.yml

```diff
@@ -7,6 +7,7 @@ sudo: false
 cache: pip
 
 addons:
+  postgresql: "9.6"
```

Comment:
Could you make this in a significant (not fixup) commit ?

Reply:
Yes let me me the whole thing green first and then I will squash together the commits for travis